_studio/mfx_lib/shared/include/mfx_h264_encode_struct_vaapi.h

```diff
@@ -243,6 +253,7 @@ typedef struct tagENCODE_CAPS
             UINT    NoESS                       : 1;
             UINT    Color420Only                : 1;
             UINT    ICQBRCSupport               : 1;
+            UCHAR   FrameToleranceMode          : 2;
```

Comment:
1) Please move to CodingLimits2. Let's keep proper alignment within unions.
2) Please rename to "FrameSizeTolerance".
3) Do we really need 2 bits? Is it possible that driver supports one tolerance mode and doesn't support another?